fact-bounty-client/src/components/AppRouter.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component, Fragment } from "react";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+
+import PrivateRoute from "./common/PrivateRoute";
+import Landing from "./containers/Landing";
+import Register from "./containers/Register";
+import Login from "./containers/Login";
+import Dashboard from "./containers/Dashboard";
+import Posts from "./containers/Posts";
+import About from "./containers/About";
+import NavBar from "./common/NavBar";
+
+class AppRouter extends Component {
+  render() {
+    return (
+      <Router>
```

Comment:
https://github.com/scorelab/fact-Bounty/blob/f570ab4c70ccc2661fb97fd138bec8d31afb0072/fact-bounty-client/src/components/AppRouter.js#L16-L28

I think it would be better if this was kept in the App itself for the clarity.

Reply:
I put is separately because this file might grow as the project grows, and it will be easy to maintain